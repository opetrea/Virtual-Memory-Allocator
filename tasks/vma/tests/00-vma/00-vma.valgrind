==91819== Memcheck, a memory error detector
==91819== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91819== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91819== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91819== Parent PID: 91818
==91819== 
==91819== 
==91819== HEAP SUMMARY:
==91819==     in use at exit: 0 bytes in 0 blocks
==91819==   total heap usage: 11 allocs, 11 frees, 8,416 bytes allocated
==91819== 
==91819== All heap blocks were freed -- no leaks are possible
==91819== 
==91819== For lists of detected and suppressed errors, rerun with: -s
==91819== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
