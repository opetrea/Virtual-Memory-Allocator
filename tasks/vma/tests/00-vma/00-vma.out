Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Total memory: 0xA bytes
Free memory: 0x9 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x1 - 0x2
Miniblock 1:		0x1		-		0x2		| RW-
Block 1 end
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
